src/common/transformations/src/transformations/common_optimizations/matmul_split_decomposition.cpp

```diff
@@ -0,0 +1,206 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/matmul_split_decomposition.hpp"
+
+#include <cstdint>
+#include <limits>
+#include <memory>
+#include <openvino/core/rt_info.hpp>
+#include <openvino/pass/pattern/op/or.hpp>
+#include <openvino/pass/pattern/op/wrap_type.hpp>
+#include <transformations/utils/utils.hpp>
```

Comment:
Use `""` for these includes.

Reply:
Removed duplicate files and replaced `<openvino/pass/pattern/op/or.hpp>` with `"openvino/pass/pattern/op/or.hpp"`